### PR TITLE
README: Update link to the current production CI

### PR DESCRIPTION
**Pull request by jmarrero**
Need to find the way to get access and update this PR with that info too.

### CODE REVIEW

**Review by cgwalters — APPROVED** [review]
Thanks!  This looks sane to me

File: README.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ workflow. For more information on getting started, see
 [HACKING](HACKING.md).
 
 The production instance is running in
-[CentOS CI](https://jenkins-fedora-coreos.apps.ocp.ci.centos.org)
+[Fedora CI](https://jenkins-fedora-coreos-pipeline.apps.ocp.fedoraproject.org/)
```
Comment (dustymabe) [inline README.md:15]:
```suggestion
an [OpenShift cluster](https://jenkins-fedora-coreos-pipeline.apps.ocp.fedoraproject.org/) in Fedora's infrastructure
```

**Review by dustymabe — APPROVED** [review]

**Comment by jmarrero** [comment]
@dustymabe @jlebon went thru our chats and did not see any specific ticket we had to create to give me permission to the infra. I guess my centos Infra permissions were passed to the new cluster. Do we still need that step? Is there a new ticket for people starting from scratch, or people just reach out to us?

**Comment by dustymabe** [comment]
@jmarrero - right now it's a PR to https://pagure.io/fedora-infra/ansible/blob/main/f/playbooks/openshift-apps/fedora-coreos-pipeline.yml and a re-running of the playbook from batcave: `sudo rbac-playbook -l os_control openshift-apps/fedora-coreos-pipeline.yml`

**Review by dustymabe — APPROVED** [review]
LGTM

**Pull request merged**